server/methods/read/channels.js

```diff
@@ -12,9 +12,8 @@ Meteor.methods({
     check(channel, Channel);
  
     images = [];
-    for (var i = 0; i < 16; i++) {
-      images = images.concat(Modules.server.googleImageSearchSync(channel.get('query')));
-    }
+    images = _.union(images, Modules.server.googleImageSearchSync(channel.get('query')));
+
     return images;
```

Comment:
Can we just change this to `return Modules.server.googleImageSearchSync(channel.get('query')));` ?